java/client/src/main/java/glide/api/models/MessageQueue.java

```diff
@@ -0,0 +1,31 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models;
```

Comment:
why is this in models?

Reply:
are we going to expose the queue to the end-user?

Reply:
I forgot to remove it, thanks.
As discussed, a user should have no queue access.